tests/foreman/cli/test_host.py

```diff
@@ -1644,6 +1645,35 @@ def test_positive_apply_errata(self):
 
     @tier3
     @run_only_on('sat')
+    def test_positive_apply_security_erratum(self):
+        """Apply security erratum to a host
+
+        @id: 4d1095c8-d354-42ac-af44-adf6dbb46deb
```

Comment:
Tokens now changes to `:` in master. Please replace 

`@id: 4d1095c8-d354-42ac-af44-adf6dbb46deb`

to

`:id: 4d1095c8-d354-42ac-af44-adf6dbb46deb`

Also for other tokens

NOTE: This change is only for master, other branches still uses the `@`